### ISSUE TITLE
Production worker docker image missing email templates folder

### ISSUE THREAD

**Issue by makoncline** [graphile/starter#76]
When running the production worker docker image, the following error occurs when an email is sent.
```
[worker(worker-2405044886759662)] ERROR: Failed task 30 (send_email) with error ENOENT: no such file or directory, open '/app/@app/worker/dist/../templates/verify_email.mjml' (17.34ms):
  Error: ENOENT: no such file or directory, open '/app/@app/worker/dist/../templates/verify_email.mjml'
```
This is fixed by placing the following on line 51 of production.Dockerfile
```
COPY --from=builder /app/@app/worker/templates/ /app/@app/worker/templates/
```

but, is there a way to only include the template folder if the $TARGET is worker?

**Comment by benjie** [comment]
> but, is there a way to only include the template folder if the $TARGET is worker?


Not AFAIK, the multipurpose builds in Docker are a bit sucky. Please raise a PR with this line added :pray:

**Comment by benjie** [comment]
Fixed in #77
